app/scripts/templates/settings.mustache

```diff
@@ -1,38 +1,34 @@
-<div id="main-content" class="card">
-  <header>
-    <h1 id="fxa-settings-header">{{#t}}Manage account{{/t}}</h1>
+<div id="fxa-settings-header-wrapper">
+  <header id="fxa-settings-header">
+    <h1 id="fxa-manage-account">{{#t}}Manage account{{/t}}</h1>
+    {{#showSignOut}}
+    <button id="signout" class="settings-button secondary">{{#t}}Sign out{{/t}}</button>
+    {{/showSignOut}}
   </header>
+  <div class="settings-success-wrapper">
+    <div class="success settings-success">Hello</div>
+  </div>
+</div>
 
-  <section>
-    <div class="error"></div>
-    <div class="success"></div>
-
-    <div class="avatar-wrapper avatar-view">
+<div id="fxa-settings-profile-header">
+    {{#avatarLinkVisible}}
+    <div class="avatar-wrapper avatar-settings-view">
+      <a href="/settings/avatar/change" class="change-avatar"></a>
     </div>
-    <p class="change-avatar-text links">
-      <a href="/settings/avatar/change">{{#t}}Change{{/t}}</a>
+    {{/avatarLinkVisible}}
+    {{^avatarLinkVisible}}
+      <a href="/settings/avatar/change" class="change-avatar">
+        <div class="avatar-wrapper avatar-settings-view"></div>
+      </a>
+    {{/avatarLinkVisible}}
```

Comment:
do we need the 2 statements above? as per as per https://github.com/mozilla/fxa-content-server/pull/2748/files#r37025438 we can remove the logic to check 

Avatar change is available to all now, so we might be able to simplify this.